### DISCUSSION TITLE
Can't get access to GitHub education edition

### DISCUSSION THREAD

**Discussion by BirdPolitician** [community/community#53654] [GitHub Education]
I've been trying to get access to GitHub education benefits for a while now, but my application keeps getting rejected, without explaining why. I am currently enrolled in year 11 at a school Github recognises, with a school email, my school email is the one I'm using, I'm using my real name, and I'm using my current Student ID as my proof of enrollment, any ideas?

**Comment by samus-aran** [comment]
Hi @BirdPolitician 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
